api/Document.json

```diff
@@ -4320,10 +4322,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "10.1"
```

Comment:
This is `document.exitPointerLock()`, added much later than most of the things in this PR, but seems right in terms of when that API was introduced.